cypress/e2e/send/send_chain_mock.cy.ts

```diff
@@ -0,0 +1,204 @@
+import { BN, bnToAvaxC } from '@c4tplatform/camino-wallet-sdk'
+
+describe('Send: C to C transfer by already owned balance', () => {
+    beforeEach(() => {
+        cy.loginWalletWith('privateKey')
+
+        cy.get('[data-cy="wallet_transfer"]').click().should('have.class', 'router-link-active')
+
+        // RPC aliases
+        cy.intercept('**/ext/bc/C/rpc').as('apiBaseFee')
+
+        // Hidden Show Breakdown
+        cy.get('.breakdown_toggle').first().click()
+
+        // Get Own C-Chain Balance
+        cy.get('.alt_breakdown > :nth-child(1) > :nth-child(6)')
+            .invoke('text')
+            .then((balance) => {
+                cy.wrap(balance.replace(/\sCAM/g, '')).as('ownChainBalance')
+            })
+
+        // Switch Source Chain to C
+        cy.get('.lists > div:nth-child(1) > .chain_select > button:nth-child(2)').click()
+
+        // Input More than Own Amount
+        cy.get('.evm_input_dropdown > .col_in > .col_big_in > input').as('inputAmount')
+        cy.get<string>('@ownChainBalance').then((balance) => {
+            const increaseBalance = parseFloat(balance) + 1
+            cy.get('@inputAmount')
+                .type(`${increaseBalance}{enter}`)
+                .then((input) => {
+                    const amount = parseFloat(input.val() as string)
+                    expect(amount).to.lte(parseFloat(balance))
+                })
+        })
+    })
+
+    it('as an error message is shown for inputting x-chain wallet address', () => {
+        cy.get<string>('@ownChainBalance').then((balance) => {
+            if (balance > 0) {
+                // Input C Chain Addr
+                cy.get('.bottom_tabs > .chain_select > button:nth-child(1)').click()
+                cy.get('[data-cy="wallet_address"]').invoke('text').as('chainAddress')
+                cy.get<string>('@chainAddress').then((chainAddr) => {
+                    cy.get('input[class="pk_in"]').eq(0).type(chainAddr)
+                })
+    
+                // Click Confirm Btn
+                cy.get('.button_primary').eq(0).click()
+    
+                cy.get('.err')
+                    .invoke('text')
+                    .should(
+                        'eq',
+                        'Invalid C Chain address. Make sure your address begins with "0x" or "C-0x"'
+                    )
+            }
+        })
+    })
+
+    it('as an error message is shown for inputting p-chain wallet address', () => {
```

Comment:
This test looks incorrect. Seems really similar to the one above, this one should check p-chain input (based on it's description), and it checks error message about C-chain address.